homeassistant/components/tuneblade_remote/entity.py

```diff
@@ -0,0 +1,31 @@
+"""TuneBladeEntity base class."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, NAME
+from .coordinator import TuneBladeDataUpdateCoordinator
+
+
+class TuneBladeEntity(CoordinatorEntity):
+    """Base entity for TuneBlade devices, including master hub."""
+
+    def __init__(
+        self,
+        coordinator: TuneBladeDataUpdateCoordinator,
+        config_entry,
+        device_id: str | None = None,
+        device_name: str | None = None,
+    ) -> None:
+        """Initialize entity with coordinator, config entry, and optional device info."""
+        super().__init__(coordinator)
+        self.config_entry = config_entry
+        self.device_id = device_id or "master"
+        self.device_name = device_name or "Master"
```

Comment:
why do we store the full config entry here?

Reply:
Not sure. Removed